api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -232,6 +216,12 @@ model PlatformProfile {
   issuerUrl: url;
 }
 
+/** OpenShift cluster image registry */
+model ClusterImageRegistryProfile {
+  /** The state of the OpenShift cluster image registry */
```

Comment:
This is also immutable. Once set cannot be modified. 
e.g you can only disable in cluster image registry during install. If you do that, you can't enable post install.
Vice versa is true as well. If enabled during install time, it can't be disabled post install time.

Reply:
This actually sent me down a rabbit hole over whether the `@visibility` decorator in TypeSpec cascades or not, because I _do_ have the `clusterImageRegistry` field above tagged as create-only.  It appears _not_ to cascade, although TypeSpec documentation is surprisingly unclear about it.